release/models/policy-forwarding/openconfig-pf-forwarding-policies.yang

```diff
@@ -518,4 +527,441 @@ submodule openconfig-pf-forwarding-policies {
+            config false;
+            description
+              "State parameters relating to IP encapsulation headers.";
+
+            uses pf-encap-ip-config;
+          }
+        }
+
+        container ipv6 {
+          when "../config/type = 'oc-types:IPV6'";
+          description
+            "Container of nodes for IPv6 encapsulation.  When this
+            container is used, an IPv6 packet with no transport header
+            is added to the encapsulation list.";
+
+          container config {
+            description
+              "Config parameters relating to IP encapsulation headers.";
+
+            uses pf-encap-ip-config;
+          }
+
+          container state {
+            config false;
+            description
+              "State parameters relating to IP encapsulation headers.";
+
+            uses pf-encap-ip-config;
+          }
+        }
+
+        container mpls {
+          when "../config/type = 'oc-types:MPLS'";
+          description
+            "Container of nodes for MPLS encapsulation.";
+
+          container config {
+            description
+              "Config parameters relating to MPLS encapsulation headers.";
+
+            uses pf-encap-mpls-config;
+          }
+
+          container state {
+            config false;
+            description
+              "State parameters relating to MPLS encapsulation headers.";
+
+            uses pf-encap-mpls-config;
+          }
+        }
+
+        container udp-v4 {
+          when "../config/type = 'oc-types:UDP'";
+          description
+            "Container of nodes for UDP in IPv4 encapsulation.  When this
+            container is used, an IPv4 header with a UDP header is added
+            to the encapsulation list.";
+
+          container config {
+            description
+              "Config parameters relating to UDP in IPv4 encapsulation
+              headers.";
+
+            uses pf-encap-udp-config;
+          }
+
+          container state {
+            config false;
+            description
+              "State parameters relating to UDP in IPv4 encapsulation
+              headers.";
+
+            uses pf-encap-udp-config;
+          }
+        }
+
+        container udp-v6 {
+          when "../config/type = 'oc-types:UDP'";
+          description
+            "Container of nodes for UDP in IPv6 encapsulation.  When this
+            container is used, an IPv6 header with a UDP header is added
+            to the encapsulation list.";
+
+          container config {
+            description
+              "Config parameters relating to UDP in IPv6 encapsulation
+              headers.";
+
+            uses pf-encap-udp-config;
+          }
+
+          container state {
+            config false;
+            description
+              "State parameters relating to UDP in IPv6 encapsulation
+              headers.";
+
+            uses pf-encap-udp-config;
+          }
+        }
+
+        container vxlan {
```

Comment:
I am not sure there is a use case for setting vxlan encapsulation using policy forwarding?  (we left it out of gribi as well)

Reply:
Ok, I have no specific use case for this in policy forwarding. Removed it.